features/admin.organizations.v1/__tests__/add-organization-modal.test.tsx

```diff
@@ -40,8 +40,8 @@ const addComponentModalProps: AddOrganizationModalPropsInterface = {
 };
 
 describe("UTC-1.0 - [Organization Management Feature] - Add Organization Modal", () => {
-    const addOrganizationMock = jest.spyOn(api, "addOrganization");
-    const getOrganizationsMock = jest.spyOn(api, "getOrganizations");
+    const addOrganizationMock: any = jest.spyOn(api, "addOrganization");
```

Comment:
See if `jest.SpyInstance` works here?

```suggestion
    const addOrganizationMock: : jest.SpyInstance = jest.spyOn(api, "addOrganization");
```

Reply:
Addressed with 3607cda